scripts/zkai_stwo_statement_envelope_benchmark.py

```
@@ -0,0 +1,744 @@
+    _check_artifact_bindings(envelope, proof)
+    external_verify(proof)
+
+
+def verify_proof_only(
+    envelope: dict[str, Any],
+    external_verify: Callable[[dict[str, Any]], None] = stwo_verify,
+) -> None:
+    external_verify(stwo_proof_payload(envelope))
+
+
+def classify_error(message: str) -> str:
```

Ruff (0.15.12)

[warning] Too many return statements (11 > 6)

(PLR0911)